ert_gui/tools/run_analysis/run_analysis_tool.py

```diff
@@ -66,19 +66,26 @@ def run(self):
             self._report_empty_target()
             return
 
-        success = analyse(self.ert, target, source)
+        try:
+            analyse(self.ert, target, source)
+            error = None
+        except Exception as e:
+            error = str(e)
```

Comment:
Could consider doing:

```suggestion
        except ErtRunError as e:
            error = str(e)
        except Exception as e:
            error = f"Uknown exception occured with error: str(e)"
```